netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -187,6 +187,12 @@ public final NettyChannelBuilder flowControlWindow(int flowControlWindow) {
     return this;
   }
 
+  @Deprecated // Use maxInboundMessageSize instead
```

Comment:
`// Use maxInboundMessageSize instead` can be put into the `@deprecated` tag in  javadoc.
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@deprecated